crates/sc-piece-cache/src/lib.rs

```diff
@@ -15,7 +15,7 @@ use subspace_core_primitives::{FlatPieces, Piece, PieceIndex, PieceIndexHash, PI
 use subspace_networking::ToMultihash;
 
 /// Defines maximum segments number stored in the cache (as pieces).
-pub const MAX_SEGMENTS_NUMBER_IN_CACHE: u64 = 10;
+pub const MAX_SEGMENTS_NUMBER_IN_CACHE: u64 = 32; // ~ 1 GB
```

Comment:
Actually, you could have had math expression here, something like `1024 * 1024 *1024 / PLOT_SIZE`, then comment is not necessary and would adjust to changing plot size automatically.

Reply:
Yeah. It seems I will need such a formula to provide a minimum for the CLI parameter.

Reply:
Let's have CLI in bytes though, it is easier to reason about for users.
We use a library in farmer to parse human-readable formats like `1G` and `500MiB`.

Reply:
Sure. I made changes in the farmer recently and plan to implement a similar conversion here.